packages/vitest-plugin-rsc/src/index.ts

```diff
@@ -40,11 +61,13 @@ export default function vitestPluginRSC(): Plugin[] {
                   "react-dom/client",
                   "react/jsx-runtime",
                   "react/jsx-dev-runtime",
-                  "@vitejs/plugin-rsc/vendor/react-server-dom/server.browser",
+                  // TODO: browser build is missing features and breaks action bind
                   "@vitejs/plugin-rsc/vendor/react-server-dom/server.edge",
                   "@vitejs/plugin-rsc/vendor/react-server-dom/client.edge",
-                  "@vitejs/plugin-rsc/vendor/react-server-dom/client.browser",
+                  // "@vitejs/plugin-rsc/vendor/react-server-dom/server.browser",
+                  // "@vitejs/plugin-rsc/vendor/react-server-dom/client.browser",
```

Comment:
It looks like browser build is actually missing some features as seen in `playground/rsc-vitest-demo/src/action-bind/server.test.tsx` failure.